cli/src/main.rs

```diff
@@ -290,11 +292,16 @@ struct Test {
 
 #[derive(Args)]
 #[command(alias = "publish")]
-/// Increment the version of a grammar
+/// Display or increment the version of a grammar
 struct Version {
-    #[arg(num_args = 1)]
     /// The version to bump to
-    pub version: SemverVersion,
+    #[arg(long_help = "When present, the version to bump to\n\
+                       When absent with no flags present, displays the current version\n\
+                       When absent with the --bump flag present, bumps the current version")]
```

Comment:
Is that still correct? If so, should specify _how_ the version is bumped.

(Also, the description is a bit confusing, since the option makes no sense without the `--bump` flag. So the second line is impossible?)

Reply:
Would it help to put examples into this help string?  Something like:

```
tree-sitter version: display the current version
tree-sitter version <version>: bump to specified version
tree-sitter version --bump <level>: automatic bump, see below for options
```

Reply:
Or maybe with less verbosity...

```
No parameters: display the current version
<version> only: bump to specified version
--bump <level>: automatic bump, see below for options
```
?

Reply:
I pushed an update...  It now renders the long help as:

![image](https://github.com/user-attachments/assets/0cc45114-ab95-4269-ad47-5ad27723ca65)

Reply:
I'm going to drop the "see below for options", and change "Bump the version" to "Automatically bump from the current version"